Cargo.toml

```diff
@@ -25,7 +25,7 @@ cfg-if = "0.1"
 compiler_builtins = { version = "0.1", optional = true }
 core = { version = "1.0", optional = true, package = "rustc-std-workspace-core" }
 
-[target.'cfg(any(unix, target_os = "redox"))'.dependencies]
+[target.'cfg(unix)'.dependencies]
 libc = { version = "0.2.62", default-features = false }
```

Comment:
We should update this to `0.2.63` to get https://github.com/rust-lang/libc/commit/e1a417f1b2f3055fdbd1054d1835a9071fb7c964

Reply:
`libc v0.2.63` was yanked due to [a serious bug](https://github.com/rust-lang/libc/issues/1515#issuecomment-531003188). VxWroks support will only be included in (currently unreleased) v0.2.64, so there is no reasons to update it now.

Reply:
Sounds good, I guess we'll have to wait for `libc v0.2.64`. Not a huge deal.